extensions/wikia/UserLogin/UserLogin.setup.php

```diff
@@ -1,4 +1,4 @@
-<?php
+	<?php
```

Comment:
I am going to remove this whitespace here.  It can be passed with the response, along with the headers.  It will prevent any HTTP-related code from being executed later.